src/main/java/by/shop/Speak.java

```diff
@@ -0,0 +1,5 @@
+package by.shop;
+
+public abstract class Speak {
```

Comment:
?????????
Again you are using a verb as a class.
Remove it.
Can you explain why did you use that?

Reply:
Rename with Dialogue.

Reply:
No. Eliminate it.
Dialogue is unneeded